designs/virtualization/4.6/empty-states/empty-states.md

```diff
@@ -0,0 +1,54 @@
+# Virtualization Default Empty States
+Foday Kargbo - UXD
+## Background
+Virtualization is being introduced to the console. Virtualization will be accessed via the main left navigation panel. Once the user navigates to Virtualization, they will see 2 tabs, one for the Virtual Machines screen, and one for the Virtual Machine Templates screen. The Virtual Machines tab will be selected by default.
+## Objectives
+Users who have not created virtual machines or virtual machine templates will not be able to see any data on these screens. 
+TBD: Do we want to create an empty state screen for the virtualization dashboard?
+
+The goals of this task:
+- Create empty state screens when there is no data to show
+- Inform the user about the current state
+- Educate the user about what they will see when data is available
+- Guide the user to take action
+- Explore any useful content that might help the user be contextually productive?
+## 1. Virtual machine default empty state screen
+![Virtual machines empty state screen](img/virtual-machines-empty-state.png "Virtual machines empty state screen")
```

Comment:
Great! I like the wording, we'll go with that. I agree that it should reflect or draw attention to the filtered view. I'm curious to know which case would this be? In other words, how is the filter determined for someone who has not created any VMs? We can adjust the wording based on this logic or we could go with a standard copy as your first suggestion. Thanks!

Reply:
I wonder if this would be dynamic when they are on all projects. We'd want to ensure if we go this route that it can detect that and adjust accordingly.

Reply:
I'll investigate this with the developer team.

Reply:
@matthewcarleton I discussed this with Tomas and it is possible to detect which filter is on and can be implemented. We can change the text dynamically to reflect the active filter. I recommend we alternate the copy dynamically to add context to the user's experience.

Confirming that for "all projects" we will use this copy:

    No virtual machines
    There are no virtual machines available. Create a virtual machine to continue.

and when filtered to a specific project we will use this copy:

No virtual machines
There are no virtual machines available in this project. Create a virtual machine to continue.

I appreciate the input @matthewcarleton @andybraren

Reply:
+1 @matthewcarleton I think it makes it clearer when you add 'across all projects' as oppose to not mentioning where.
And maybe we can just add the word 'Please' before the 'Create a virtual machine to continue'. IMO it makes it sound more pleasant :)

Reply:
@yfrimanm Yes, something we can consider. Thanks for the input!